backend/middlewares/validators/lessonValidators.ts

```diff
@@ -27,9 +28,44 @@ export const lessonRequestDtoValidator = async (
   if (!validatePrimitive(body.image, "string")) {
     return res.status(400).send(getApiValidationError("image", "string"));
   }
-  // TO DO: Write better validator for content
+  // content validation
+  if(!validateContentList(body.content)){
+    return res.status(400).send("There was an error in the lesson content.")
```

Comment:
Could we have a better error message specifying that the inputs didn't fit type expectations?

Reply:
I agree with this. Knowing what part of the content failed will probably help us down the line